site/src/install.sh

```diff
@@ -9,7 +9,7 @@ case $(uname -s) in
 Darwin) target="macos" ;;
 *) target="linux" ;;
 esac
-
+https://github.com/alecks/pylon-cli/pull/1
```

Comment:
Your clipboard works.

Reply:
Thanks for the kind words. I caused it to not work now